<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>129</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>굴림</family>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>manas</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
	border: 3px solid #000000;
	background: rgba(100, 100, 100, 125);
	pading: 10px;
	font -size: 18px;
	font: 굴림;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#lineEdit_2
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
#lineEdit_2:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>저장하고자하는경로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
QPushButton:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
   <property name="text">
    <string>다운로드</string>
   </property>
  </widget>
  <widget class="QLabel" name="msg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>415</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>15</horstretch>
     <verstretch>15</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>링크 읽기</string>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>84</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#pushButton_2
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
#pushButton_2:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
   <property name="text">
    <string>끄 기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>84</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#pushButton_3
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
#pushButton_3:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
   <property name="text">
    <string>최 소 화</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
#combo
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
#combo:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
#combo_2
{
	border: 3px solid #000000;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	font -size: 18px;
	border-radius: 5px;
}
#combo_2:hover
{
	border: 3px solid #aaaaaa;
	background: #3a7999;
	color: #f2f2f2;
	pading: 10px;
	border-radius: 5px;

}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>combo</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>set_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>mana_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>file_load_check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>gui_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>win_min()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo</sender>
   <signal>activated(QString)</signal>
   <receiver>Form</receiver>
   <slot>select_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_dir()</slot>
  <slot>mana_down()</slot>
  <slot>file_load_check()</slot>
  <slot>gui_close()</slot>
  <slot>win_min()</slot>
  <slot>select_item()</slot>
 </slots>
</ui>
